rpc-api/cmd/notify-rpc-api/grpc.go

```diff
@@ -0,0 +1,447 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/govau/notify/rpc-api/notify"
```

Comment:
Move to end and alias as `pb`:

```go
import (
  "bytes"

  "google.golang.org/grpc"

  pb "github.com/govau/notify/rpc-api/notify"
)
```

This is so that you know when you see `notify.Service` that you are referring to proto code. Especially important if one uses a `notify` package at the domain level because otherwise you'd have a conflict.

E.g. https://github.com/AusDTO/tell-us-once/blob/master/tuo/coci/order.go

Reply:
if we create a notify package we can rename this. but we won't, because we'll be deleting this entire file.